<launch>

  <include file="$(find um7)/launch/um7_pub_and_display.launch"> </include>
  
  <node pkg="imu_visualization" type="Coordinate_system_correction.py" name="Coordinate_system_correction" output="screen">
	<param name="imu_data_topic_name" value="/sensor/imu/um7/data" />
	<param name="mag_data_topic_name" value="/sensor/imu/um7/magfield_msg" />
    <param name="imu_name" value="CHRobotics_imu" />
    <param name="package_path" value="$(find imu_visualization)/configs/" /> -->
	<!-- <param name="mag_data_vector_topic_name" value="/sensor/imu/xsens_mti/mag_calib" /> -->
  </node>

</launch>
